appcelerator-archive/amp
# Emit JSON default values for gRPC gateway

Closes #1275 

Made the gRPC gateway sending default values.

Since the gRPC gateway tests are not working on travis, here is a way of testing:

## How to test
```
$ ampmake build
$ alias amp='amp -s localhost'
$ amp cluster create -t local
$ amp user signup --name user --password password --email user@email.amp
$ amp login --name user --password password
$ export TOKEN=$(cat $HOME/.config/amp/localhost.credentials)
$ curl -s -H "Authorization: amp $TOKEN" -H "Content-Type: application/json" -X POST -d '{"name":"name","data":"data"}' http://gw.local.appcelerator.io/v1/dashboards | jq
{
  "dashboard": {
    "id": "7298a5cbe5b93062fed047e0b28f6b44db90c579e2920277f9d3a72cb40d625e",
    "name": "name",
    "owner": {
      "type": "USER",
      "name": "user"
    },
    "create_dt": "1495914526",
    "data": "data"
  }
}
```

The field `dashboard.owner.type` should be populated with `USER`.
